### PR TITLE
Hotfix github migration

### PR DESCRIPTION
**Pull request by sgreilich**
Getting my head around the way git works.

### CODE REVIEW

**Comment by sgreilich** [comment]
I have no idea, but it would not let me create a pull request to merge with develop, but only with master (which is not necessarily in accordance with the model). If that because I branched from master (by mistake)?

**Comment by grzanka** [comment]
Have you been creating pull request from web interface of from application ?

File: README_DEVELOPERS [hunk 1]
```diff
@@ -146,16 +146,14 @@ Please refrain from doing so.
 
 Basic instruction on the use of GNU autotools are found in INSTALL.
 
-The libamtrack sources can be checkout from it svn repository at sourceforce.net by:
+The libamtrack sources can be fetched from GIT repository at github.com by:
 
 
-svn co https://svn.code.sf.net/p/libamtrack/code/trunk libamtrack
+git clone https://github.com/libamtrack/library.git
```
Comment (grzanka) [inline README_DEVELOPERS]: Nice code ;)

**Comment by sgreilich** [comment]
I used the application...

Sendt fra min iPhone

Den 04/09/2015 kl. 11.09 skrev grzanka <notifications@github.com<mailto:notifications@github.com>>:


Have you been creating pull request from web interface of from application ?

**Pull request merged**